src/components/Heroes/columns.jsx

```diff
@@ -23,72 +24,128 @@ export default {
     displayName: strings.hero_pick_ban_rate,
     field: 'pickBanRatePro',
     sortFn: true,
-    displayFn: (row, col, field) => percentileDisplay(field, row.matchCountPro),
+    displayFn,
+    relativeBars: {
```

Comment:
Would it be cleaner to have relativeBars just accept a function that returns the decimal value?   It can assume a divisor of 1